notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb

```diff
@@ -0,0 +1,655 @@
+{
```

Comment:
<p>Those plots are really nice. My only suggestion here would be to add a summary guiding the reader to the conclusion, based on the hypothesis raised above of what a "PASSED" test means. I think this would close the loop on what you want to show, also making the reader's life easier. Please, do the same for the "FAILED" example below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
I added a clarifying sentence for each outcome. Let me know what you think.